imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import unicode_literals
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+import uuid
+from django.utils.translation import ugettext as _
+
+# Create your models here.
+
+
+class ImageProfileManager(models.Manager):
```

Comment:
Might be worth naming this something more descriptive, like `ImageProfileActivityManager`